docs/index.md

```diff
@@ -594,9 +594,16 @@ The `random_reference` property creates a reference to a random, existing row fr
       random_reference: Owner
 ```
 
-The selected row can be any one that matches the object type and was already created in the current iteration of the recipe. For example, this previous recipe was executed twenty times (iterations) to generate 200 `Pets` and `Owners`. The selected rows in the first iteration would be one of the first ten `Owners`, and the selected rows in the last iteration would be one of the last ten.
+The selected row can be any one that matches the object type (or nickname) and was already created in the current iteration of the recipe. For example, this previous recipe was executed twenty times (iterations) to generate 200 `Pets` and `Owners`. The selected rows in the first iteration would be one of the first ten `Owners`, and the selected rows in the last iteration would be one of the last ten.
```

Comment:
This whole section is confusing. I think what it's saying is it pulls a random row by object, but the object has to be created for it to exist (and get selected).  And if a recipe for 10 pets and owners gets executed 20 times, the pets that get assigned to owners created during the same iteration. But I'm not sure. However, the change that's being made (adding "(or nickname)" seems fine, so it doesn't make sense to block on this.